site/src/pages/index.tsx

```diff
@@ -86,17 +87,21 @@ export default ({ location }) => {
               </a>
             </div>
             <div className={styles.guidelinesTextContainer}>
-              <div className={styles.heading}>
-                <a href={withPrefix("/guidelines/overview")}>Guidelines</a>
+              <div className={styles.headingContainer}>
```

Comment:
`headingContainer should be able to replaced with the upcoming `Spacing` component in future